src/main/java/org/wikipedia/dataclient/mwapi/MwQueryPage.java

```diff
@@ -144,6 +146,14 @@ public void appendTitleFragment(@Nullable String fragment) {
         @NonNull public String timeStamp() {
             return StringUtils.defaultString(timeStamp);
         }
+
+        public String getRevisionId() {
+            return revisionId;
+        }
+
+        public String getUser() {
```

Comment:
Please annotate these return values as `@Nullable` if you're OK with getting a null value, or make it `@NonNull` and return `StringUtils.defaultString(user)`.